doc/Tex/RAM_SCB_install.tex

```diff
@@ -1,180 +1,72 @@
-
-\item{\href{http://www.unidata.ucar.edu/downloads/netcdf/ftp/netcdf-4.0.1.tar.gz}{http://www.unidata.ucar.edu/downloads/netcdf/ftp/netcdf-4.0.1.tar.gz}}
-\end{enumerate}
-
-
-Unpack the tarballs in the usual fashion:
-\begin{verbatim}
-tar -xzvf netcdf-4.0.1.tar.gz
-\end{verbatim}
-
-Because these libraries are not owned or maintained by the authors of RAM-SCB, 
-only minimal installation instructions are provided here.  The reader is 
-encouraged to explore the websites and help files associated with each package.
-
-
-\subsubsection{Ncarg Quick Install}
-Installation of the NCAR Graphics library from source is an involved process.  
-It is strongly suggested that the user first check their system to see if the 
-library is already installed or install via binary.
-
-\subsubsection{NetCDF Quick Install}
-Unpack the NetCDF tarball, {\tt cd} into the unzipped directory, and use the 
-following commands:
-\begin{verbatim}
-./configure --prefix=/Users/ram_user/netcdf_lib
-make install
-\end{verbatim}
-This will install the NetCDF library in {\tt /Users/ram\_user/netcdf\_lib} 
-(the home directory for user \textit{ram\_user}); change the value of 
-{\tt --prefix} to select the installation destination.  For installation, 
-NetCDF requires only a C compiler.  {\tt gcc} is strongly suggested.
-
-\subsubsection{Pspline Quick Install}
-Pspline requires NetCDF, so be sure that NetCDF is correctly installed before 
-continuing.  After unpacking the tarball, compile the library using
-\begin{verbatim}
-make FORTRAN_VARIANT=Portland NETCDF_DIR=~/netcdf_lib
-\end{verbatim}
-It is important to specify the Fortran compiler that coincides with the 
-compiler you will use to make the RAM-SCB executable.  Other options accepted 
-by the {\tt FORTRAN\_VARIANT} variable are {\tt PathScale},  {\tt GCC}(gfortran
-and g95), {\tt NagWare},  {\tt Fujitsu}, {\tt Intel}, and {\tt Absoft}.  
-Ensure that the value of {\tt NETCDF\_DIR} coincides with the installation 
-directory of NetCDF.
-
-{\tt make} may fail during the creation of the Pspline test utilities.  If 
-this happens, check for the creation of a {\tt lib} directory and several 
-\verb*l*.al files contained within.  If they exist, disregard the error 
-message and continue to the next step.
-
-Install Pspline as follows (changing the value of {\tt PREFIX} to select 
-installation directory):
-\begin{verbatim}
-make install PREFIX=~/psline_portland
-\end{verbatim}
-On machines where you will use different compilers, it is helpful to add the 
-compiler name to the name of the installation directory.
+% QUESTION: All of the information in this section is out of date. Even the NETCDF installation instructions are incorrect given that they have split it into a C version and a Fortran version. This is still easy enough to describe and walk through, but the installation of GSL from source isn't as easy. Do we really want to have this section?
```

Comment:
I'm not convinced that our documentation needs to have install instructions for dependencies. My preference would be to say what the dependencies are, point to where they are (e.g., unidata for NetCDF), and note that everything needs to be compiled with the same compiler. If there are gotchas - like NetCDF-C and NetCDF-Fortran having different version numbers - then we should probably note that.

Where we will want more detailed instructions is in making sure that RAM-SCB finds the libraries.

Reply:
That sounds good to me, especially since we don't have anything weird like PSpline now